//todo put to use
export function handleError(error:any) {
    console.log('An error occurred', error)
}